resources/api/pierone-api.yaml

```diff
@@ -239,6 +241,7 @@ paths:
           in: body
           schema:
             description: "the manifest"
+            type: string
```

Comment:
It's JSON, but

* the structure of v1/v2 schema are quite different
* we expect v2 to change again anyways (when it also has a signature)
* we don't really care.

Fine by me for now ^^